src/Generator/SubProfileGenerator.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Drupal\lightning\Generator;
+
+use Drupal\Console\Core\Generator\Generator;
+
+class SubProfileGenerator extends Generator {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function generate(
```

Comment:
I think the arguments to generate should be on one line, and type hinted where possible.